CONTRIBUTING.md

```diff
@@ -29,3 +29,92 @@ Recommendations and requirements for how to best contribute to Parquet. We striv
 ### License
 By contributing your code, you agree to license your contribution under the terms of the APLv2:
 https://github.com/apache/parquet-format/blob/master/LICENSE
+
+### Additions/Changes to the Format
+
+The general steps for adding features to the format are as follows:
```

Comment:
Is it worth leading this off with something to the effect that minor changes to the spec (fixing typos, providing clarification) can be made at the discretion of the committers without the need for a more formal process? Or is that just obvious?

Reply:
It can't hurt to be explicit.

Reply:
This is incorporated.